### PR TITLE
[MI-1677] collapsible order list

### PR DESCRIPTION
**Pull request by kcasas**
/cc @zendesk/mintegrations

### Description
* Added `Accordion` component to hold logic for collapsing order panels. Caret added to control collapse all.
* Added `resizeApp` side effect. Implemented when orders was rendered and when order panels are collapsed and uncollapsed.
* Additional attribute `rel="noopener noreferrer"` on links with `target="_blank"`. To avoid security [issue](https://mathiasbynens.github.io/rel-noopener/).

### References
* JIRA: https://zendesk.atlassian.net/browse/MI-1677

### Risks
* [low] might destroy rendering of elements

### CODE REVIEW

File: src/styles/common.scss [hunk 1]
```diff
@@ -2,6 +2,22 @@
...
+         -o-transition-timing-function: ease;
+            transition-timing-function: ease;
+    -webkit-transition-duration: .35s;
+         -o-transition-duration: .35s;
+            transition-duration: .35s;
+    -webkit-transition-property: height, visibility;
+         -o-transition-property: height, visibility;
+            transition-property: height, visibility;
+  }
+
```
Comment (kcasas) [inline src/styles/common.scss]: Copied from the latest [bootstrap.css](https://maxcdn.bootstrapcdn.com/bootstrap/latest/css/bootstrap.css). `react-bootstrap` needs the latest css in order for collapse transitions to work properly. Cannot use the latest version of `bootstrap.css` as it changes the look of the app.

File: webpack.config.js [hunk 2]
```diff
@@ -55,12 +55,6 @@ module.exports = {
   },
   plugins: [
     extractStyles,
-    new webpack.optimize.UglifyJsPlugin({
-      compress: {
-        drop_debugger: false,
-        warnings: false,
-      },
-    }),
```
Comment (kcasas) [inline webpack.config.js]: Removed to improve debugging experience while in development. Uglifying still works on `yarn build`. Also, by removing this we also dropped the debugger on production builds.

File: src/Containers/AccordionContainer.js [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+import Accordion from '../Components/Orders/Accordion';
+
+const mapStateToProps = () => {};
+
+const mapDispatchToProps = dispatch => Object.assign(
+  { dispatch },
+  bindActionCreators({ resizeApp }, dispatch),
+);
+
+const AccordionContainer = connect(mapStateToProps, mapDispatchToProps)(Accordion);
```
Comment (samgavinio) [inline src/Containers/AccordionContainer.js]: ❤️

File: tests/Components/Orders/Accordion/Accordion.test.js [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+import ReactTestUtils from 'react-dom/test-utils';
+import Panel from 'react-bootstrap/lib/Panel';
+import Accordion from '../../../../src/Components/Orders/Accordion';
+import Order from '../../../../src/Components/Orders/Order';
+import { renderer, renderIntoDocument } from '../../../i18n';
+
+describe('<Accordion />', () => {
+  const orders = faker.shopify.orders();
+  const elem = (
+    <Accordion resizeApp={jest.fn()}>
```
Comment (samgavinio) [inline tests/Components/Orders/Accordion/Accordion.test.js]: Asserting that the `resizeApp` prop is called would be useful.

**Pull request merged**